Adding documentation for specifying writing log files to disk.

justinrmiller opened the pull request:
This PR resolves: https://github.com/qdrant/landing_page/issues/1125

Provides documentation around writing Qdrant log files to disk.

I tried to build the project locally to verify but saw the following when I ran `hugo serve -D`:

```
ERROR TOCSS-DART: failed to transform "/css/main.scss" (text/x-scss): "/Users/justin/Development/Projects/landing_page/qdrant-landing/themes/qdrant-2024/assets/css/_bootstrap-custom.scss:2:8": Can't find stylesheet to import.
Built in 1744 ms
Error: error building site: TOCSS-DART: failed to transform "/css/vendor/splide.scss" (text/x-scss): "/Users/justin/Development/Projects/landing_page/qdrant-landing/themes/qdrant-2024/assets/css/vendor/splide.scss:1:8": Can't find stylesheet to import.
```

Anush008 commented:
Thanks @justinrmiller 

> I tried to build the project locally to verify but saw the following when I ran `hugo serve -D`:
> 
> ```
> ERROR TOCSS-DART: failed to transform "/css/main.scss" (text/x-scss): "/Users/justin/Development/Projects/landing_page/qdrant-landing/themes/qdrant-2024/assets/css/_bootstrap-custom.scss:2:8": Can't find stylesheet to import.
> Built in 1744 ms
> Error: error building site: TOCSS-DART: failed to transform "/css/vendor/splide.scss" (text/x-scss): "/Users/justin/Development/Projects/landing_page/qdrant-landing/themes/qdrant-2024/assets/css/vendor/splide.scss:1:8": Can't find stylesheet to import.
> ```

You'll need to install https://sass-lang.com/install/.

bashofmann reviewed (changes requested):
I don't think this should go into the Hybrid Cloud configuration. Logging to disk does not make much sense in the context of Hybrid Cloud which runs on Kubernetes. Apart from the data volumes, the storage in the container is ephemeral. And Kubernetes has a well established logging system, which relies on Qdrant's default to log to stdout.

Since this a configuration value of the core database, it should go to here: https://qdrant.tech/documentation/guides/configuration/

justinrmiller commented:
> I don't think this should go into the Hybrid Cloud configuration. Logging to disk does not make much sense in the context of Hybrid Cloud which runs on Kubernetes. Apart from the data volumes, the storage in the container is ephemeral. And Kubernetes has a well established logging system, which relies on Qdrant's default to log to stdout.
> 
> Since this a configuration value of the core database, it should go to here: https://qdrant.tech/documentation/guides/configuration/

Thanks for the feedback, I've moved the documentation of the feature to the configuration page and added a note on which `log_level` parameter takes precedence.

bashofmann commented on qdrant-landing/content/documentation/guides/configuration.md:
```diff
@@ -91,6 +91,18 @@ QDRANT__SERVICE__HTTP_PORT=1234 ./qdrant
 log_level: INFO
 
+# Logging to disk
+# You can write logs to disk by specifying a log file. 
```
```suggestion
# By default, Qdrant logs to stdout. You can write logs to disk by specifying a log file. 
```

bashofmann commented:
Small addition, but otherwise looks good. @timvisee If I remember correctly, since this is in the configuration yaml block, this also needs to be added to `qdrant/qdrant`?

justinrmiller commented:
> Small addition, but otherwise looks good. @timvisee If I remember correctly, since this is in the configuration yaml block, this also needs to be added to `qdrant/qdrant`?

Thanks, updated. I went ahead and created a PR against `qdrant/qdrant` to add this to `config.yaml`. 

https://github.com/qdrant/qdrant/pull/5041

timvisee reviewed (changes requested):
Thank you for the suggestion. Since more and more people seem to be asking about this I'm fine with adding the stubs for it in the configuration file.

@bashofmann You're right on adding this in core first, thanks for the hint.

I propose to settle on something in <https://github.com/qdrant/qdrant/pull/5041>. Then we can update this PR in the same manner. I'll mark it as 'request changes' for the time being.

timvisee reviewed (changes requested):
@justinrmiller Could you apply the same here as in <https://github.com/qdrant/qdrant/pull/5041>?

justinrmiller commented:
> @justinrmiller Could you apply the same here as in [qdrant/qdrant#5041](https://github.com/qdrant/qdrant/pull/5041)?

Yep updated.
